Week3/hanoi/src/Hanoi.java

```diff
@@ -8,13 +8,16 @@ private void moveDisk(int diskNumber, int startingPoint, int destination) {
   }
 
   // Returns the answer of Hanoi problem with given numDisks.
+  // numDisks: number of disks to be moved.
+  // Disks should be move from starting point bar to destionation bar.
```

Comment:
plz write more specific such as Calculate the step of moving the number of given disks from the given starting point to the destination .......